README.md

```diff
@@ -2,6 +2,8 @@
 
 This is a simple visualizer for [Kaitai Struct](https://github.com/kaitai-io/kaitai_struct) project.
 
+![](https://user-images.githubusercontent.com/8781107/97787079-f8fe0580-1bc0-11eb-92de-bcdb40d3f407.png)
```

Comment:
1. Let's have a screenshot inside the repo, it's a bad idea to store it outside.
2. Please don't neglect `[]` — that goes into `img alt`, let's not forget about bots, people what use screen readers, people without images loading, etc.

Reply:
Fixed. PTAL.